react/features/base/conference/middleware.js

```diff
@@ -77,6 +80,35 @@ function _connectionEstablished(store, next, action) {
 }
 
 /**
+ * Does extra sync up on properties that may need to be updated, after
+ * the conference was joined.
+ *
+ * @param {Store} store - The Redux store in which the specified action is being
+ * dispatched.
+ * @param {Dispatch} next - The Redux dispatch function to dispatch the
+ * specified action to the specified store.
+ * @param {Action} action - The Redux action CONFERENCE_JOINED which is being
+ * dispatched in the specified store.
+ * @private
+ * @returns {Object} The new state that is the result of the reduction of the
+ * specified action.
+ */
+function _conferenceJoined(store, next, action) {
+    const result = next(action);
+    const { audioOnly, conference }
+        = store.getState()['features/base/conference'];
+
+    // FIXME On Web the audio only mode for "start audio only" is toggled before
+    // conference is added to the redux store ("on conference joined" action)
+    // and the LastN value needs to be synchronized here.
+    if (audioOnly && !conference.getLastN() !== 0) {
```

Comment:
Yes it relies on the fact that audio only mode adjusts last n, but when starting in audio only the conference is not in redux yet (on web) and it just silently does nothing